appmap-java-maven-plugin/README.md

```diff
@@ -0,0 +1,67 @@
+AppLand AppMap Maven Plugin for Java
+--------------------------------
+
+
+- [Building](#building)
+- [Agent Configuration](#agent-configuration)
+- [Maven Plugin Configuration](#maven-plugin-config)
+    - [Plugin Goals](#plugin-goals)
+    - [Plugin configuration options](#plugin-configuration)
+    - [Example](#example)
+
+
+# Building
+Artifacts will be written to `target/` use `appmap-java-plugin-[VERSION].jar`. as your maven plugin.
+```bash
+$ mvn clean install
+```
+
+# Agent Configuration
+When you run your program, the agent reads configuration settings from `appmap.yml` by default.
+
+Please read configuration options from [AppMap Java Agent README.md](../README.md)
+
+# Maven Plugin Configuration
+
+## Plugin goals
+prepare-agent : adds appmap.jar to JVM execution as javaagent
+
+## Plugin configuration options
+outputD.directory (default: ./target/appmap/)
+configF.file (default: ./appmap.yml)
```

Comment:
Is this meant to be `output.directory`? Some other configuration options seem to have a similar problem. `event.valueSize`, `config.file`?

Reply:
oh yes, there was an error copying from a google doc with suggested changes, let me fix that real quick.